### PR TITLE
Fix/bugs - Discord Tickets

### PR DESCRIPTION
**Pull request by QU35T-code**
# Description

Fixes for problems that users have encountered.

# Related issues

`Discord ticket 0093` -> Fix Sliver ARM build
`Discord ticket 0088` -> Add objectwalker
`Discord ticket 0084` -> Add ldapsearch
`Discord ticket 0083` -> Add rsync
`Discord ticket 0073` -> Install openjdk-11-jdk and openjdk-17-jdk for ghidra
`Discord ticket 0101, 0082, 0090, 0092` -> Fixed ruby ​​environment different from tmux suspending wpscan installation (for testing)

Fix `ysoserial` link from https://jitpack.io/com/github/frohoff/ysoserial/master-SNAPSHOT/ysoserial-master-SNAPSHOT.jar to https://github.com/frohoff/ysoserial/releases/latest/download/ysoserial-all.jar

Fixed installing `pth-tools` for x86 (AMD) with `libreadline8` issue.

Fixed `checksec` installation by adding `lief` dependency.

### CODE REVIEW

File: sources/assets/zsh/aliases.d/pth-tools [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
-alias pth-rpcclient='LD_LIBRARY_PATH="${LD_LIBRARY_PATH:+$LD_LIBRARY_PATH:}/opt/tools/pth-toolkit/lib/:/opt/tools/pth-toolkit/lib/private/" /opt/tools/pth-toolkit/bin/rpcclient'
-alias pth-smbclient='LD_LIBRARY_PATH="${LD_LIBRARY_PATH:+$LD_LIBRARY_PATH:}/opt/tools/pth-toolkit/lib/:/opt/tools/pth-toolkit/lib/private/" /opt/tools/pth-toolkit/bin/smbclient'
-alias pth-smbget='LD_LIBRARY_PATH="${LD_LIBRARY_PATH:+$LD_LIBRARY_PATH:}/opt/tools/pth-toolkit/lib/:/opt/tools/pth-toolkit/lib/private/" /opt/tools/pth-toolkit/bin/smbget'
-alias pth-winexe='LD_LIBRARY_PATH="${LD_LIBRARY_PATH:+$LD_LIBRARY_PATH:}/opt/tools/pth-toolkit/lib/:/opt/tools/pth-toolkit/lib/private/" /opt/tools/pth-toolkit/bin/winexe'
-alias pth-wmic='LD_LIBRARY_PATH="${LD_LIBRARY_PATH:+$LD_LIBRARY_PATH:}/opt/tools/pth-toolkit/lib/:/opt/tools/pth-toolkit/lib/private/" /opt/tools/pth-toolkit/bin/wmic'
-alias pth-wmis='LD_LIBRARY_PATH="${LD_LIBRARY_PATH:+$LD_LIBRARY_PATH:}/opt/tools/pth-toolkit/lib/:/opt/tools/pth-toolkit/lib/private/" /opt/tools/pth-toolkit/bin/wmis'
```
Comment (Dramelac) [inline sources/assets/zsh/aliases.d/pth-tools:7]: Why did you remove the variable ?

Comment (QU35T-code) [reply sources/assets/zsh/aliases.d/pth-tools:7]: Because the variable is empty and the best is to use the libraries which are in the folder `/opt/tools/pth-toolkit/lib/` and `/opt/tools/pth-toolkit/lib/private` to avoid conflicts ?

```bash
root@exegol-test /workspace # echo $LD_LIBRARY_PATH

```
